packages/git.rb

```diff
@@ -54,6 +55,6 @@ def self.postinstall
 
   def self.check
     # Skip several t9010-svn-fe and t9011-svn-da tests since they fail.
-    #system "GIT_SKIP_TESTS='t9010.16 t9010.20 t9011.1[49] t9011.2[0346] t9011.31 ' #{@make_cmd} test"
+    system "GIT_SKIP_TESTS='t9010.16 t9010.20 t9011.1[49] t9011.2[0346] t9011.31 ' #{@make_cmd} test"
```

Comment:
Skipping tests because they fail is not a good way to fix the issues.  You might as well not test at all.

Reply:
OK. I didn't write this.

Reply:
But you uncommented it.  How do you know it doesn't fail on arm or i686?